drape/stipple_pen_resource.hpp

```diff
@@ -22,7 +22,7 @@ uint32_t constexpr kMaxStipplePenLength = 512;  /// @todo Should be equal with k
 // Based on ./data/patterns.txt, the most of patterns have 2 entries (4 entries for triangles pattern).
 using PenPatternT = buffer_vector<uint16_t, 2>;
 
-inline constexpr uint16_t PatternFloat2Pixel(double d)
+inline uint16_t PatternFloat2Pixel(double d)
```

Comment:
Well, yes, constexpr here is since C++23

Reply:
Did the generated code change for modern arm64 compilers? If it became slower, it may be better to put constexpr under an ifdef.

Reply:
There are no usages of this function in constexpr context.